discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/FileRegistryServiceImpl.java

```diff
@@ -37,6 +37,7 @@ public class FileRegistryServiceImpl implements RegistryService<ConfigChangeList
     private static final Configuration CONFIG = ConfigurationFactory.getInstance();
     private static final String POSTFIX_GROUPLIST = ".grouplist";
     private static final String ENDPOINT_SPLIT_CHAR = ";";
+    private static final String ENDPOINT_AGAIN_SPLIT_CHAR = ",";
```

Comment:
AGAIN -> old ?   if necessary ?

Reply:
It doesn't seem necessary now, and if a change is to be made in this regard, it should be resubmitted as a separate pr

Reply:
done